aws/resource_aws_fsx_lustre_file_system_test.go

```diff
@@ -785,6 +786,7 @@ resource "aws_fsx_lustre_file_system" "test" {
   import_path      = "s3://${aws_s3_bucket.test.bucket}"
   storage_capacity = 1200
   subnet_ids       = [aws_subnet.test1.id]
+  deployment_type  = "SCRATCH_2"
```

Comment:
Nit: Our test configurations tend to only set "what's necessary" for the purposes of making assertions easier to understand and validate our `Default` assumptions. In these cases it might be clearer for future readers (especially those without GovCloud knowledge or access) to see exactly why this argument value is being set, e.g.

```
  deployment_type  = data.aws_partition.current.partition == "aws-us-gov" ? "SCRATCH_2" : null # GovCloud does not support SCRATCH_1
```